analysis/report_templates/experimental.html

```diff
@@ -18,20 +18,31 @@
 
 {% block top_level_ranking %}
 
-<h5>Rank by median on benchmarks, then by average rank (Critical difference)</h5>
-{{ super() }}
+<h5>Rank by median on benchmarks, then by avereage normalized score</h5>
+<div class="row">
+    <div class="col s7 offset-s3">
+        {{ experiment.rank_by_median_and_average_normalized_score.to_frame().to_html() }}
+    </div>
+</div>
 
-<h5>Rank by pair-wise statistical test wins on benchmarks, then by average rank</h5>
+<h5>Rank by median on benchmarks, then by average rank</h5>
```

Comment:
Do we need both mean and median? I vaguely remember we concluded mean was better because the distributions are nonnormal.

Reply:
The idea was to put everything here just in case. But it might be too redundant, let me remove it.